### PR TITLE
Update TipCalc to MvvmCross 4.2.0

### PR DESCRIPTION
**Pull request by tofutim**
All platforms tested. Preparing to add .Mac version.

### CODE REVIEW

File: TipCalc/TipCalc.UI.Droid/TipCalc.UI.Droid.csproj [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
-    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.4</TargetFrameworkVersion>
```
Comment (Cheesebaron) [inline TipCalc/TipCalc.UI.Droid/TipCalc.UI.Droid.csproj]: Why downgrade target? Why not just compile with latest?

Comment (tofutim) [inline TipCalc/TipCalc.UI.Droid/TipCalc.UI.Droid.csproj]: I realized why I keep downgrading to 4.4. In the default Xamarin install,
two images are installed of KitKat machines, a 5" and a 7". Without going
through downloading of the latest images, you end up with 4.4 which is why
Xamarin is setting my TargetFrameworkVersion back.

Is there a reason to compile to the latest?

**Comment by tofutim** [comment]
I'm comfortable to going back to 6.0 if desired.

**Comment by Cheesebaron** [comment]
Target framework has nothing to do with minimum or maximum API levels. Don't mess with them.

**Comment by tofutim** [comment]
I didn't mess with the target framework  - it was Xamarin that did it on my
machine.

On Mon, Jun 20, 2016 at 12:10 AM, Martijn van Dijk <notifications@github.com

**Pull request merged**